package.json

```diff
@@ -42,40 +42,40 @@
     "babel-loader": "^8.0.6",
     "babel-plugin-transform-es2015-arrow-functions": "^6.22.0",
     "babel-plugin-transform-es2015-shorthand-properties": "^6.24.1",
-    "chromedriver": "^75.0.0",
+    "chromedriver": "^75.0.1",
     "copy-webpack-plugin": "^5.0.3",
     "core-js": "3.1.4",
     "css-loader": "^3.0.0",
     "cucumber": "cucumber/cucumber-js#issue-727-retry",
     "cucumber-pretty": "^1.5.2",
     "easygettext": "^2.7.0",
-    "eslint": "5.16.0",
+    "eslint": "6.0.1",
```

Comment:
Build when running `yarn watch-all` still fails because of the eslint.

Reply:
let's nail eslint to < 6 - THX - will take care

Reply:
updated ...

Reply:
Works fine now, thanks!